net-irc/ngircd/ngircd-17.1.ebuild

```diff
@@ -2,14 +2,13 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Header: /var/cvsroot/gentoo-x86/net-irc/ngircd/ngircd-17.1.ebuild,v 1.5 2014/03/10 10:58:31 ssuominen Exp $
 
-EAPI=4
+EAPI="5"
```

Comment:
You should rather avoid raising EAPI in stable ebuilds. Just in case the change could trigger some conditional change in eclass, and break stuff.